api/libretimeapi/models.py

```diff
@@ -0,0 +1,663 @@
+# This is an auto-generated Django model module.
+# You'll have to do the following manually to clean this up:
+#   * Rearrange models' order
```

Comment:
Yeah, I was thinking that. Though, because of how django works, it would be things like libretimeapi/models/smartblocks.py

Reply:
I was thinking of splitting stuff into multiple modules via `INSTALLED_APPS` (would also need `__init__.py`, `app.py` next to `models.py` in each subdir). I'm not sure if it's worth the effort for now though.

Reply:
Hmm... I'm not sure what advantages that gives us. How well separated are the different components?

Reply:
It would make the `models.py` files smaller. It also helps isolating different concerns from each other. It's also probably overkill for now and we can still refactor to a multi-app setup later if we run into issues.

Reply:
I've broken `models.py` out into multiple files under the same app